packages/api-client/src/main/user/data/UserPushRemoveData.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Wire
+ * Copyright (C) 2019 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+export interface UserPushRemoveData {
+  app: string;
+  client: string;
+  token: string;
+  transport: 'GCM';
```

Comment:
Is it always GCM? what does it mean?

Reply:
GCM = Google Cloud Messaging

Reply:
So far I only saw 'GCM' but reading the [PushTokens spec](https://github.com/wearezeta/backend-api-docs/wiki/API-User-PushTokens#transport) there can be also `GCM`, `APNS`, `APNS_SANDBOX`, `APNS_VOIP` & `APNS_VOIP_SANDBOX`. I will add them too. 👍